server/svix-server/src/core/types.rs

```diff
@@ -427,6 +428,20 @@ impl Validate for EventTypeNameSet {
     }
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct RetrySchedule(pub Vec<u64>);
+json_wrapper!(RetrySchedule);
+
+impl RetrySchedule {
+    pub fn as_durations(&self) -> Vec<Duration> {
+        self.0
+            .clone()
```

Comment:
We can remove this clone if we are ok consuming self. Consuming self happens to work out fine in this PR as its implemented, but left it this way in case it could be useful in the future. Happy to remove.